.github/workflows/pr-test-rust.yml

```
@@ -321,6 +354,54 @@ jobs:
+            reruns: "--reruns 2 --reruns-delay 5"
+            parallel_opts: ""
+            test_filter: ""
+            setup_vllm: 'true'
+            ignore_opts: ""
+    runs-on: ${{ matrix.runner || 'k8s-runner-gpu' }}
+    timeout-minutes: ${{ matrix.timeout }}
+    steps:
+      - uses: ./.github/actions/run-e2e
+        with:
+          setup_vllm: ${{ matrix.setup_vllm }}
+          setup_trtllm: ${{ matrix.setup_trtllm }}
```

actionlint (1.7.11)

[error] property "setup_trtllm" is not defined in object type {env_vars: string; extra_deps: string; ignore_opts: string; name: string; parallel_opts: string; reruns: string; runner: string; setup_vllm: bool; test_dirs: string; test_filter: string; timeout: number; upload_benchmarks: bool}

(expression)